structuredefinitions/UKCore-DiagnosticReport.xml

```diff
@@ -27,6 +27,15 @@
   <baseDefinition value="http://hl7.org/fhir/StructureDefinition/DiagnosticReport" />
   <derivation value="constraint" />
   <differential>
+    <element id="DiagnosticReport">
+      <path value="DiagnosticReport" />
+      <constraint>
+        <key value="ukcore-diag-lab-002" />
+        <severity value="error" />
+        <human value="Only one of DiagnosticReport.media.link or DiagnosticReport.media.extension:mediaLinkR5 can be populated at a time." />
+        <expression value="media.empty() or extension('http://hl7.org/fhir/5.0/StructureDefinition/extension-DiagnosticReport.media.link').empty()" />
```

Comment:
The word 'link' needs removing from both lines, so should be:

<human value="Only one of DiagnosticReport.media or DiagnosticReport.media.extension:mediaR5 can be populated at a time." />
        <expression value="media.empty() or extension('http://hl7.org/fhir/5.0/StructureDefinition/extension-DiagnosticReport.media').empty()" />